### DISCUSSION TITLE
Welcome to python-backend Discussions!

### DISCUSSION THREAD

**Discussion by bh0717-dev** [bh0717-dev/python-backend#11] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wonderBH717** [comment, marked as answer]
You have to install module to compile this.

**Reply by bh0717-dev** [reply]
Thanks

**Comment by bh0717-dev** [comment]
I can't compile this project

**Reply by wonderBH717** [reply]
You need to change the version of python
